Dfe.PrepareConversions/Dfe.PrepareConversions/Utils/ProjectListHelper.cs

```diff
@@ -131,7 +131,7 @@ public static string MapPerformanceDataHint(string schoolType)
          return sType switch
          {
             "pupil referral unit" => $"Your document will automatically include some Ofsted inspection data. Educational performance data isn't published for pupil referral units.\r\n\r\nAsk the pupil referral unit to share their educational performance and absence data with you. You can add that to the document once you have created it.",
-            _ => "This information will be added to your project document automatically."
+            _ => "This information with not be added to your project document."
```

Comment:
will not?

Reply:
Because, we no longer displaying educational performance data in preview and project template and we have already removed information via another PR.  It's be advised by Mike. If people needs to see the information, people will access it through excel sheet and will determine later, how people will access info.

Reply:
Ok, just need to change with -> will then and all good :)